internal/broker/export_test.go

```diff
@@ -105,5 +107,12 @@ func (b *Broker) FetchUserInfo(sessionID string, cachedInfo *authCachedInfo) (st
 		return "", err
 	}
 
-	return b.userInfoFromClaims(uInfo, groups)
+	userInfo := b.userInfoFromClaims(uInfo, groups)
+	data, err := json.Marshal(userInfo)
+	if err != nil {
+		return "", err
+	}
+	var indented bytes.Buffer
+	err = json.Indent(&indented, data, "", "\t\t")
```

Comment:
Do we really need two tabs?

Reply:
No, I did it to be consistent with what we had in the current golden files, so to reduce the diffing for making the review easier.